server/ai_live_video.go

```diff
@@ -393,12 +409,27 @@ func startEventsSubscribe(ctx context.Context, url *url.URL, params aiRequestPar
 				continue
 			}
 
-			var event map[string]interface{}
-			if err := json.Unmarshal(body, &event); err != nil {
+			var eventWrapper struct {
+				QueueEventType string                 `json:"queue_event_type"`
+				Event          map[string]interface{} `json:"event"`
+			}
+			if err := json.Unmarshal(body, &eventWrapper); err != nil {
 				clog.Infof(ctx, "Failed to parse JSON from events subscription: %s", err)
 				continue
 			}
 
+			event := eventWrapper.Event
+			queueEventType := eventWrapper.QueueEventType
+			if event == nil {
+				// If no "event" field found, treat the entire body as the event
```

Comment:
You can add that it's for backward compatibility and after some time we can remove it.